src/proto/FunctionRpc.proto

```diff
@@ -370,3 +371,8 @@ message RpcHttp {
   bool enable_content_negotiation= 16;
   TypedData rawBody = 17;
 }
+
+//Used to send the dependency management request to the language worker
+message DependencyManagementRequest {
```

Comment:
Is this supposed to be part of `StreamingMessage.content`?

Reply:
That's right @peterhuene , I'll add it to StreamingMessage

Reply:
@peterhuene - It is optional for a language worker to implement new message types. Functions host will check for capabilities sent in WorkerInitResponse  before sending this message.